doc/classes/Basis.xml

```diff
@@ -7,7 +7,7 @@
 		A 3×3 matrix used for representing 3D rotation and scale. Usually used as an orthogonal basis for a [Transform3D].
 		Contains 3 vector fields X, Y and Z as its columns, which are typically interpreted as the local basis vectors of a transformation. For such use, it is composed of a scaling and a rotation matrix, in that order (M = R.S).
 		Basis can also be accessed as an array of 3D vectors. These vectors are usually orthogonal to each other, but are not necessarily normalized (due to scaling).
-		For more information, read the "Matrices and transforms" documentation article.
+		For a general introduction, see the "[url=$DOCS_URL/tutorials/math/matrices_and_transforms.html]Matrices and transforms[/url]" tutorial.
```

Comment:
We don't usually put links in quotes, the link styling should make it stand out already.